doc/getting-started.md

```diff
@@ -75,15 +75,15 @@ export class Users {
 
 Here's what we've changed. We've:
 
-1. Swapped out `HttpClient` with `Endpoint`.
-2. Altered the `@inject()` decorator to use the `Endpoint` resolver.
-3. Completely removed the config calls in the constructor. (We added that in `src/main.js` earlier).
-4. Assigned the endpoint to the viewModel.
-5. Changed `this.http.fetch('users')` to `this.githubEndpoint.find('users')`. Notice that we removed the `.json()` step, too.
+1 Swapped out `HttpClient` with `Endpoint`.
```

Comment:
meant additional.
but hmm. seems i broke the listing. wierd was sure i checked that.

anyways, it's first about whether md linting would be usefull or not